conda-environment.yml

```diff
@@ -0,0 +1,21 @@
+name: rose-dev
+channels:
+  - conda-forge
+dependencies:
+  # outut: metomi-rose-base
+  - __unix
+  - python >= 3.7.0
+  - aiofiles
+  - jinja2 >=2.10.1
+  - keyring >=23,<24
+  - ldap3
+  - metomi-isodatetime >=1!3.0,<1!4.0
+  - psutil >=5.6.0
+  - requests
+  - sqlalchemy >=1,<2
+  - importlib_metadata >=5.0  # py<3.12 TODO: selectors don't work for noarch packages
+  - importlib_resources >=2.0  # py<3.9 TODO: selectors don't work for noarch packages
+  # - tornado (rosie disco is temporally disabled)
+
+  # output: metomi-rose
+  - pygraphviz >1.0,!=1.8
```

Comment:
@astroDimitrios 

Once this PR has been merged, please rebase your branch and add the new GUI dependencies to the bottom of this file.